package/com.unity.formats.usd/Runtime/InitUsd.cs

```diff
@@ -64,12 +65,11 @@ public static bool Initialize()
         // USD has several auxillary C++ plugin discovery files which must be discoverable at run-time
         // We store those libs in Support/ThirdParty/Usd and then set a magic environment variable to let
         // USD's libPlug know where to look to find them.
-        private static void SetupUsdPath()
+        private static void SetupUsdPath([CallerFilePath] string sourceFilePath = "")
         {
 #if UNITY_EDITOR
-            // TODO: this is not robust, e.g. if anyone changes CWD from the default, package resolution
-            // will fail. Following up with UPM devs to see what we can do about it.
-            var supPath = System.IO.Path.GetFullPath("Packages/com.unity.formats.usd/Runtime/Plugins");
+            var fileInfo = new System.IO.FileInfo(sourceFilePath);
```

Comment:
Tested by renaming the package ?

Reply:
Yep, I renamed the package folder and updated the manifest to see if it was working allright.